Client/mods/deathmatch/logic/lua/CLuaArgument.cpp

```diff
@@ -446,6 +517,29 @@ void CLuaArgument::Push(lua_State* luaVM, CFastHashMap<CLuaArguments*, int>* pKn
                 lua_pushlstring(luaVM, m_strString.c_str(), m_strString.length());
                 break;
             }
+
+            case LUA_TFUNCTION:
+            {
+                if (m_pResource)
+                {
+                    lua_newtable(luaVM);            // create callable
+
+                    lua_pushresource(luaVM, m_pResource);            // push function inside callable
```

Comment:
When a referenced resource become stopped this pointer will turn into the dangling one. From what I was able to see there is no an invalidation mechanism for functions.

Reply:
i agree

Reply:
i have some cleaner and better version in my mind for this
i will work on it and update its PR to use that instead